iowrite.cpp

```diff
@@ -1075,6 +1075,12 @@ bool writeIonosphereGridMetadata(vlsv::Writer& vlsvWriter) {
   } else {
     vlsvWriter.writeArray("MESH", xmlAttributes, 0, 1, ionosphereGridElementsAndCorners.data());
   }
+  
+  // Write different parameters of the class Ionosphere into the VLSL file
```

Comment:
Oh and this set of writing calls could be put behind an if check that we actually are running an ionosphere, let me dig up an example for you.